### PR TITLE
Add new module CustomProviders, HanaOnAzure, MonitoringSolutions, ConnectedK8s, and K8sConfiguration

### PR DESCRIPTION
**Pull request by isra-fel**
## Description

Modules: 
- Az.MonitoringSolutions
  - Operations of Microsoft first party solutions.
- Az.CustomProvider
  - Operations of custom resource provider and association.
- Az.HanaOnAzure
  - Operations of SAP monitor and provider instance.

## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [x] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: src/HanaOnAzure/HanaOnAzure.sln [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29709.97
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Accounts", "..\Accounts\Accounts\Accounts.csproj", "{142D7B0B-388A-4CEB-A228-7F6D423C5C2E}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Az.Databricks", "Az.Databricks.csproj", "{B6E9156B-F29F-41BE-A5E4-533A873C99B8}"
```
Comment (dolauli) [inline src/HanaOnAzure/HanaOnAzure.sln:7]: Should be Az.HanaOnAzure instead of Az.Databricks

**Pull request merged**